enzyme/test/Enzyme/ForwardMode/Faddeeva_erfi.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -sroa -instsimplify -simplifycfg -adce -S | FileCheck %s
+; RUN: %opt < %s %loadEnzyme -enzyme -enzyme-preopt=false -mem2reg -instsimplify -instcombine -simplifycfg -adce -S | FileCheck %s
```

Comment:
Do we need instcombine here, if possible we should avoid because its impact is very diferent from version to version (which makes it a great optimization, but bad for a test).

Reply:
fixed